data/schemas/schemas_openxmlformats_org_wordprocessingml_2006_main.json

```diff
@@ -40521,19 +40468,23 @@
           "QName": "w:numId",
           "PropertyName": "NumberID",
           "Type": "Int32Value",
-          "PropertyComments": "Numbering Definition Instance ID",
```

Comment:
Why do you remove the descriptive `PropertyComments` and replace it with the local name of the Open XML element?

I don't know what the convention for those `PropertyComments` values is, but it looks like there is some inconsistency in those that are introduced in this commit.